airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/utils/backward_compatibility.py

```diff
@@ -90,29 +74,61 @@ def check_if_type_of_type_field_changed(self):
             # This might be something already guaranteed by JSON schema validation.
             if isinstance(change.t1, str):
                 if not isinstance(change.t2, list):
-                    self._raise_error("The current spec change a type field from string to an invalid value.")
+                    self._raise_error("The current {context} change a type field from string to an invalid value.")
```

Comment:
```suggestion
                    self._raise_error("The current {context} changed a type field from string to an invalid value.")
```

Reply:
I made the error messages more consistent